lib/logging.dart

```diff
@@ -61,6 +61,18 @@ class Logger {
     return _loggers.putIfAbsent(name, () => new Logger._named(name));
   }
 
+  /// Creates new detached Logger
```

Comment:
minor grammar nits: 

* add "a" (or replace "new" by "a") and add a period:

```dart
/// Creates a detached logger.
```

* use  `[Logger]` and `[Logger.new]` so that dartdoc tools can create links.